src/page-multi-facility/MultiFacilityImpactDashboard.tsx

```diff
@@ -5,18 +5,26 @@ import styled from "styled-components";
 import { FetchedFacilities } from "../constants";
 import { getFacilities } from "../database";
 import Colors from "../design-system/Colors";
+import { MarkColors } from "../design-system/Colors";
 import iconAddSrc from "../design-system/icons/ic_add.svg";
 import Loading from "../design-system/Loading";
 import TextLabel from "../design-system/TextLabel";
 import { useFlag } from "../feature-flags";
 import useError from "../hooks/useError";
 import useFacilitiesRtData from "../hooks/useFacilitiesRtData";
+import ChartArea from "../impact-dashboard/ChartArea";
 import { EpidemicModelProvider } from "../impact-dashboard/EpidemicModelContext";
 import {
   getFacilitiesRtDataById,
   updateFacilityRtData,
 } from "../infection-model/rt";
 import { useLocaleDataState } from "../locale-data-context";
+import { publicCurvesAndCaseToggles } from "../page-multi-facility/curveToggles";
+import {
+  useCurrentCurveData,
+  useModelInputs,
+} from "../page-response-impact/hooks";
+import { getCurveChartData } from "../page-response-impact/responseChartData";
```

Comment:
I wonder if we should extract this out of `page-response-impact` into a neutral package, since it's now being used in multiple pages. We should probably not import from one page-* into another page-* unless there's a very compelling reason.

Reply:
I moved the reused hooks to the hooks package and responseChartData to impact-dashboard.

Reply:
The current system and impact charts aren't showing up because of the existing problem with impact charts, but I could locally verify the impact charts don't have "cases" anymore (by reverting b898ee712f5351e41193a088c133811508236ea1).